content/doc/installation/with-docker.en-us.md

```diff
@@ -17,111 +17,179 @@ menu:
-
-```
-$ docker pull postgres:alpine
-$ docker run -d --name gitea-db \
-    -e POSTGRES_PASSWORD=<PASSWORD> \
-    -v gitea-db-data:/var/lib/postgresql/data \
-    -p 5432:5432 \
-    postgres:alpine
-```
-
-Now the database is up and running, we need to configure it. Make sure you remember the password for when Gitea starts.
-
-```
-$ docker exec -it gitea-db psql -U postgres
-psql (9.6.1)
-Type "help" for help.
-
-postgres=# CREATE USER gitea WITH PASSWORD '<PASSWORD>';
-CREATE ROLE
-postgres=# CREATE DATABASE gitea OWNER gitea;
-CREATE DATABASE
-postgres=# \q
-$
-```
-
-### Gitea
-
-First, the named volume
-
-```
-$ docker volume create --name gitea-data
-```
-
-Now run (remove the dns entry if not needed):
-
-```
-$ docker run -d --name gitea \
-	--link gitea-db:gitea-db \
-	--dns 10.12.10.160 \
-	-p 11180:3000 \
-	-p 8322:22 \
-	-v gitea-data:/data \
-	gitea/gitea:latest
-```
-
-You should now have two Docker containers for Gitea and PostgreSQL plus two Docker named volumes and you should have a running instance of Gitea, to access the web UI just visit http://hostname:11180/install/ in your favorite browser.
-
-In the web UI set:
-- Database Type: PostgreSQL
-- Host: gitea-db:5432
-- User: gitea
-- Password: <PASSWORD>
-- ...
-- HTTP port: 11180
-- Application URL: http://hostname:11180
-
-Apply settings. You should be able to access to web UI now and create your first user.
+This reference setup guides you through the setup based on `docker-compose`, the installation of `docker-compose` is out of scope of this documentation. To install `docker-compose` follow the official [install instructions](https://docs.docker.com/compose/install/).
+
+## Basics
+
+The most simple setup just creates a volume and a network and starts the `gitea/gitea:latest` image as a service. Since there is no database available you can start it only with SQLite3. Create a directory like `gitea` and paste the following content into a file named `docker-compose.yml`.
+
+```yaml
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
+    image: gitea/gitea:latest
+    restart: always
+    networks:
+      - gitea
+    volumes:
+      - ./gitea:/data
+    ports:
+      - 3000:3000
+      - 222:22
+```
+
+## Custom port
+
+To bind the integrated openSSH daemon and the webserver on a different port, you just need to adjust the port section. It's common to just change the host port and keep the ports within the container like they are.
+
+```diff
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
+    image: gitea/gitea:latest
+    restart: always
+    networks:
+      - gitea
+    volumes:
+      - ./gitea:/data
+    ports:
+-      - 3000:3000
+-      - 222:22
++      - 8080:3000
++      - 2221:22
+```
+
+## MySQL database
```

Comment:
For the notice for /install or editing app.ini. I was thinking of : 
Create a sub-part containing all custom DB and at start of this section specify that the config of DB should be reflected in app.ini or at /install